lib/groonga/client/request/select.rb

```diff
@@ -182,6 +192,31 @@ def initialize(request)
             @request = request
           end
 
+          # Adds a `between` condition then return a new `select`
+          # request object.
+          #
+          # @example: Multiple conditions
+          #    request.
+          #      filter.between("tags", "min", "min_border", "max", "max_border").
+          #        # -> --filter 'between(tags, min, "min_border", max, "max_border")'
+          #
+          # @example: Ignore no values case
+          #    request.
+          #      filter.between("tags")
+          #        # -> --filter ''
+          #
+          # @param column_name_or_value [String, Symbol, Integer] The target column name
+          #   or target num.
+          #
+          # @param values [Object] Range of values.
+          #
+          # @return [Groonga::Client::Request::Select]
+          #   The new request with the given condition.
+          def between(column_name_or_value, *values)
```

Comment:
Please don't use needless variable length arguments. If we can use fixed length arguments, we should use it for easy to use and easy to maintain.